templates/search_derived/search_clients_last_seen_v1/query.sql

```diff
@@ -0,0 +1,164 @@
+WITH
+  _derived_search_cols AS (
+    SELECT
+      CASE
+          WHEN engine IS NULL THEN 'none'
+          WHEN STARTS_WITH(engine, 'google') THEN 'google'
+          WHEN STARTS_WITH(engine, 'ddg') OR STARTS_WITH(engine, 'duckduckgo') THEN 'ddg'
+          WHEN STARTS_WITH(engine, 'bing') THEN 'bing'
+          WHEN STARTS_WITH(engine, 'yandex') THEN 'yandex'
+          ELSE 'other'
+      END AS short_engine,
+      COALESCE(organic, 0) + COALESCE(sap, 0) + COALESCE(unknown, 0) + COALESCE(tagged_sap, 0) + COALESCE(tagged_follow_on, 0) AS total_searches,
+      COALESCE(tagged_sap, 0) + COALESCE(tagged_follow_on, 0) AS tagged_searches,
+      COALESCE(ad_click, 0) AS ad_click,
+      COALESCE(search_with_ads, 0) AS search_with_ads,
+      * EXCEPT (ad_click, search_with_ads)
+    FROM
+      search_clients_daily_v8
+    WHERE
+      submission_date = @submission_date
+      and sample_id = "84"
```

Comment:
sample_id should be an integer here. And can we add a comment about why the magic number 84?